src/operator/delay.ts

```diff
@@ -134,7 +134,7 @@ class DelaySubscriber<T> extends Subscriber<T> {
 }
 
 class DelayMessage<T> {
-  constructor(private time: number,
-              private notification: any) {
+  constructor(public time: number,
+              public notification: Notification<T>) {
```

Comment:
The old code was wrong. These members should be public to be used in [`DelaySubscriber`](https://github.com/ReactiveX/rxjs/blob/master/src/operator/delay.ts#L82)